### PR TITLE
DS-2925 Fix Critical vulnerability

### PR DESCRIPTION
**Pull request by ildomar-grings**
## Overview

[DS-2925](https://loyaltynz.atlassian.net/browse/DS-2925)

## What is the change?
  - Bump `rack` from `2.0.5` to `2.2.7`
  - Bump `rake` from `12.3.1` to `12.3.3`
  - Bump `yard` from `0.9.15` to `0.9.34`
  - Added `bundle-audit` into CICD steps
  
## Notes

[DS-2925]: https://loyaltynz.atlassian.net/browse/DS-2925?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -7,9 +9,12 @@ matrix:
     - rvm: 2.5.1
     - rvm: 2.4.4
     - rvm: 2.3.7
-
 cache: bundler
 
+before_script:
+  - bundle exec bundle-audit update
+  - bundle exec bundle-audit check
```
Comment (nickpiercy95) [inline .travis.yml:16]: Nice 💯

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,12 @@
 # Change log
 
+## v1.3.1
```
Comment (nickpiercy95) [inline CHANGELOG.md:3]: I think this should be a minor version bump, since we're we're updating Rack from 2.0 to 2.2

```suggestion
## v1.4.0
```

**Review by nickpiercy95 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (ildomar-grings) [reply CHANGELOG.md:3]: Actually, based on the fact the application itself doesn't change, there are no new feature to consider on the counting. 

That is why I consider this update minor.

Comment (nickpiercy95) [reply CHANGELOG.md:3]: @ildomar-grings fair enough

**Review by nickpiercy95 — APPROVED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,12 @@
...
+
+- Critical vulnerabilities update [DS-2925](https://loyaltynz.atlassian.net/browse/DS-2925)
```
Comment (pond) [inline CHANGELOG.md:5]: This reference and this link are both meaningless to anyone outside of LNZL. Recommend copying details into a GitHub issue within this repo & referencing that instead.

**Review by pond — CHANGES_REQUESTED** [review]
PR is for a public repo but it includes a link to LNZL's private JIRA repository.

File: CHANGELOG.md (hunk 3 above)
Comment (ildomar-grings) [reply CHANGELOG.md:9]: Good point. Reference removed.
Thanks

**Pull request merged**